src/index.html

```diff
@@ -1,37 +1,26 @@
 <!DOCTYPE html>
```

Comment:
[OPTIONAL]
- Into your README file add instructions on how to install your project locally and if are needed additional instructions there will be nice for non-technical people or recruiters to be more easy to install your project.
- Update your live demo link.
- Add a License.